### PR TITLE
[audio] Time measurements should all be in seconds

### LINKED ISSUE #33255: [docs] New Expo Audio `seekTo` takes `milliseconds` instead of `seconds`
### Summary

Hey, as seen in the below image (and the docs page as well), the function `seekTo` seems to accept a parameter for the number of `seconds` you want to seek. 

image

However, this is incorrect and internally it actually assumes that what you pass is a millisecond. This can be tested simply by adding the below code to the example snack present in the docs. 

```tsx
<Button title="seek to 2 seconds" onPress={() => player.seekTo(2000)} />
```

This is confusing as I could not understand where am I going wrong with my implementation. 

We can either update the docs to reflect that it takes in `milliseconds` or change the logic to accept `seconds`. 

### Link to the related docs page

https://docs.expo.dev/versions/v52.0.0/sdk/audio/#seektoseconds

### Anything else?

I am up to changing whatever code/docs but yeah not sure if we should change the docs or change the logic.

### PR DESCRIPTION
**Pull request by alanjhughes**
# Why
Closes #33255
Units of time should all be in seconds. 

# How
This was not correctly reflected some of the arguments and return values. 

# Test Plan
Bare-expo. Working as before.

### CODE REVIEW

**Review by tsapeta — COMMENTED** [review]
I think we rather want to always use seconds, which is a basic SI unit. `expo-video` uses seconds, Web APIs use seconds (especially take a look at `currentTime` property)

**Comment by alanjhughes** [comment]
@tsapeta yeah, makes sense. I didn't think about web. Updated

File: packages/expo-audio/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 ### 🐛 Bug fixes
 
+- Update docs and api so all units of time should be in milliseconds, not seconds. ([#33320](https://github.com/expo/expo/pull/33320) by [@alanjhughes](https://github.com/alanjhughes))
```
Comment (tsapeta) [inline packages/expo-audio/CHANGELOG.md:11]: To be honest I have no idea how to qualify this change. Isn't it breaking?
```suggestion
- Update docs and API so all units of time are in seconds, not milliseconds. ([#33320](https://github.com/expo/expo/pull/33320) by [@alanjhughes](https://github.com/alanjhughes))
```

File: docs/public/static/data/v52.0.0/expo-audio.json
Comment (tsapeta) [inline docs/public/static/data/v52.0.0/expo-audio.json:1]: Updating this JSON on main will automatically update the docs for SDK 52, is this what you want? I think we want to publish these changes first.

Comment (alanjhughes) [reply docs/public/static/data/v52.0.0/expo-audio.json:1]: So do we only update the docs after publishing now? Not fully clear on the process now.

File: packages/expo-audio/CHANGELOG.md (hunk 1 above)
Comment (alanjhughes) [reply packages/expo-audio/CHANGELOG.md:11]: Not really. It fixes an error. The docs said it should be seconds but  I was returning milliseconds

File: docs/public/static/data/v52.0.0/expo-audio.json
Comment (tsapeta) [reply docs/public/static/data/v52.0.0/expo-audio.json:1]: In this specific case, where you only removed one comma, it doesn't really matter. But if the change is bigger, updating the docs without publishing the package could make it even more misleading for a few days, until someone publishes that. So yes, the docs data should be synchronized after publishing, i.e. something that is `/unversioned/` on `sdk-52` branch should be the same as `/v52.0.0/` on `main`.

**Review by tsapeta — APPROVED** [review]

**Pull request merged**